plugins/UsersManager/tests/Fixtures/AnonymousUser.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+/**
+ * Matomo - free/libre analytics platform
+ *
+ * @link    https://matomo.org
+ * @license http://www.gnu.org/licenses/gpl-3.0.html GPL v3 or later
+ */
+
+namespace Piwik\Plugins\UsersManager\tests\Fixtures;
+
+use Piwik\DbHelper;
+use Piwik\Plugins\UsersManager\API;
+use Piwik\Tests\Framework\Fixture;
+
+class AnonymousUser extends Fixture
```

Comment:
Could there be a better name for the fixture since it's not only about a single anonymous user?

Reply:
We could expand the fixture to be `OneUserPerRoleAndAnonymous`. The current tests should not be affected as we still would only need to check one additional user for the bulk operations.

Keeping just the current two users could be `AnonymousAndRegularUser`, sounds not more detailed though :thinking:

Reply:
Yeah, I couldn't think of any better name myself that wouldn't be too convoluted or didn't require additional changes. I guess this is ok for now.